src/ontology/mondo-ingest.Makefile

```diff
@@ -515,7 +522,7 @@ sync-subclassof-%:
 	$(MAKE) $(REPORTDIR)/$*.subclass.direct-in-mondo-only.tsv
 
 # Side effects: Deletes SOURCE.subclass.direct-in-mondo-only.tsv's from which the combination is made.
-$(REPORTDIR)/sync-subClassOf.direct-in-mondo-only.tsv: $(foreach n,$(ALL_COMPONENT_IDS), sync-subclassof-$(n))
+$(REPORTDIR)/sync-subClassOf.direct-in-mondo-only.tsv: $(foreach n,$(ALL_COMPONENT_IDS), sync-subclassof-$(n)) tmp/mondo.db
```

Comment:
This was not the problem, but since `sync_subclassof_collate_direct_in_mondo_only` depends on `tmp/mondo.db` its better to spell it out.

In general, try always to declare the inputs of a python script, dont assume presence of files etc in a python script.

Reply:
Ah, an oversight. Thanks for correcting!